definitions/component-mapping/component-mapping.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   name: m5-release-pipeline
 spec:
   params:
+    - name: ApplicationSnapshot
```

Comment:
Could you rename it so it’s clear that it’s a json?

Reply:
No - https://github.com/redhat-appstudio/release-service/blob/main/tekton/pipeline_run.go#L115

Reply:
Sorry, fat fingers. I meant the description.